lib/tasks/evm_application.rb

```diff
@@ -206,4 +206,26 @@ def self.deployment_status
     return "upgrade"        if ActiveRecord::Migrator.needs_migration?
     "redeployment"
   end
+
+  def self.queue_overview
+    output_status(queue_status)
+  end
+
+  def self.queue_status
+    MiqQueue.select(:zone, :queue_name, :role, :class_name, :method_name)
+            .select('min(coalesce(deliver_on, created_on)) as oldest')
+            .select('count(*) as count')
+            .group(1, 2, 3, 4, 5)
+            .order('1, 2, 3, 6 desc, 4, 5')
```

Comment:
I have never seen this before...is this a postgres specific thing?

Reply:
pretty sure this is ANSI SQL.

I'll see what I can do to fix